src/Microsoft.IdentityModel.Tokens/JsonWebKey.cs

```diff
@@ -25,18 +25,18 @@
 //
 //------------------------------------------------------------------------------
 
-using Microsoft.IdentityModel.Logging;
-using Newtonsoft.Json;
 using System;
 using System.Collections.Generic;
+using Microsoft.IdentityModel.Logging;
+using Newtonsoft.Json;
 
-namespace System.IdentityModel.Tokens.Jwt
+namespace Microsoft.IdentityModel.Tokens
 {
     /// <summary>
     /// Represents a Json Web Key as defined in http://tools.ietf.org/html/rfc7517.
     /// </summary>
     [JsonObject]
-    public class JsonWebKey
+    public class JsonWebKey : AsymmetricSecurityKey
```

Comment:
@PinpointTownes where the conversion takes place is what this PR is about. As Tushar mentioned the JsonWebKeySet currently does the translation. Asp.Net uses OIDCConfiguration.SigningKeys (ICollection<SecurityKey>) when validating a token. As Tushar mentioned we are trying to provide a way for Linux users to take control. It seemed to be natural in the SignatureProvider.